packages/cli/ete-tests/src/tests/validate/__snapshots__/validate.test.ts.snap

```diff
@@ -13,7 +13,8 @@ exports[`validate > no-api 1`] = `
 exports[`validate > no-generator 1`] = `"Misconfigured fern directory: please see the docs at https://buildwithfern.com/learn/api-definition/introduction/what-is-the-fern-folder"`;
 
 exports[`validate > simple 1`] = `
-"[api]: Found 2 errors and 0 warnings. Run fern check --warnings to print out the warnings.
+"[api]: ✓ All checks passed
```

Comment:
We'll need to update this - ideally the prefix is `[generators]` now that we have a separate `@fern-api/generators-validator`, but we'll need to make more changes to customize the prefix [here](https://github.com/fern-api/fern/blob/d93860839ab4c4995cb2cf5857ebb3d94d387c69/packages/cli/cli/src/cli-context/CliContext.ts#L208).

Reply:
Fixed in [this commit](https://github.com/fern-api/fern/pull/5363/commits/33d07c895d690956556472bcd45fb391720841d8).